config/locales/in_person_proofing/en.yml

```diff
@@ -47,7 +47,7 @@ en:
           city_label: City
           is_searching_message: Searching for Post Office locations…
           none_found: Sorry, there are no participating Post Offices within 50 miles of
-            %{address}.
+            %{address}
           none_found_tip: You can search using a different address, or add photos of your
```

Comment:
The [mocks](https://www.figma.com/file/I0SvsdxBvxtVvb7sPw3FSY/LG-8944%3A-PO-Search-in-the-Help-Center?type=design&node-id=5-1136&mode=design&t=sUndN2QsSKtWYcDd-0) and the French translation for none_found to not have a trailing period so I removed the period for the English and Spanish translation.

<img width="393" alt="Screenshot 2023-09-18 at 1 28 45 PM" src="https://github.com/18F/identity-idp/assets/125507397/e6dabf7c-d001-4c7e-afd8-18c5b0ea4680">

Reply:
If it hasn't already, then this verbiage change should get a cursory review by @18F/identity-joy-designers. The outcome may also affect the changes in [18F/identity-site#1167](https://github.com/18F/identity-site/pull/1167/files#diff-9cb7402d5eab4cff8fa7a110d476f2de84bd75a366aac48cba62e8a52a1d8bbeR8) since I don't see a prior design review for those changes.

Reply:
@rutvigupta-design  Here is the piece that could use a review, please and thank you.

Reply:
@gina-yamada matching the English and Spanish versions to the Figma designs makes sense to me. Screenshots look good to me!